pkg/segment/segment.go

```diff
@@ -172,3 +195,14 @@ func errorType(err error) string {
 func RunningInTerminal() bool {
 	return term.IsTerminal(int(os.Stdin.Fd()))
 }
+
+// IsTelemetryEnabled returns true if user has consented to telemetry
+func IsTelemetryEnabled(cfg *preference.PreferenceInfo) bool {
+	// The env variable gets precedence in this decision
+	if os.Getenv(DisableTelemetryEnv) == "true" {
```

Comment:
Might be a good idea to convert the env value to lowercase first

Reply:
This variable is used by 2 other files. Also, why do you suggest that?

Reply:
So that `export ODO_DISABLE_TELEMETRY=TRUE` works too. Do we want to support that?

Reply:
> So that `export ODO_DISABLE_TELEMETRY=TRUE` works too. Do we want to support that?

It would already be covered, no? If I execute an odo command with `export ODO_DISABLE_TELEMETRY=TRUE`, the `os.Getenv` call here will grab it, no? Just to be clear, you didn't miss this declaration earlier in the file, right?
```go
 const DisableTelemetryEnv = "ODO_DISABLE_TELEMETRY"
```

Reply:
> So that `export ODO_DISABLE_TELEMETRY=TRUE` works too. Do we want to support that?

Or is this about `true`, `True`, `TRUE`, etc.?

Reply:
> Or is this about true, True, TRUE, etc.?

Yes, I meant the case of `TRUE`, `True` etc